src/OrchardCore.Modules/OrchardCore.Admin/Startup.cs

```diff
@@ -45,18 +47,15 @@ public override void ConfigureServices(IServiceCollection services)
             });
 
             services.AddTransient<IAreaControllerRouteMapper, AdminAreaControllerRouteMapper>();
-
             services.AddScoped<IPermissionProvider, Permissions>();
             services.AddScoped<IThemeSelector, AdminThemeSelector>();
             services.AddScoped<IAdminThemeService, AdminThemeService>();
-
             services.AddScoped<IDisplayDriver<ISite>, AdminSiteSettingsDisplayDriver>();
             services.AddScoped<IPermissionProvider, PermissionsAdminSettings>();
             services.AddScoped<INavigationProvider, AdminMenu>();
-
-            services.Configure<AdminOptions>(_configuration.GetSection("OrchardCore.Admin"));
-
             services.AddSingleton<IPageRouteModelProvider, AdminPageRouteModelProvider>();
+
+            services.Configure<AdminOptions>(_configuration.GetSection("OrchardCore_Admin"));
```

Comment:
So if `_` is the new convention we would need to update the docs and appsettings samples

But as you want, maybe part of another doc PR ;) Just as a reminder

And we fallback to the `.` so for perf people would have to update their appsettings

**OR maybe**

Kepp the `.` convention and then just update the doc to say that under linux it is better to use a `_` separator

Reply:
I found the `docs` folder as part of this repo - will make changes there as well 👍 .